packages/js/e2e-environment/config/jest-object.config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External Dependencies
+ */
+const { E2E_RETRY_TIMES } = process.env;
+
+const setupJestRetries = () => {
+	const retryTimes = E2E_RETRY_TIMES ? E2E_RETRY_TIMES : 3;
+
+	jest.retryTimes( retryTimes );
+};
+
+const setupJestObject = () => {
```

Comment:
This function isn't necessary. `setupJestRetries()` should be added to the e2e-environment readme as well as the repository e2e readme.

Reply:
Based on our slack conversation, I've gone ahead exported both methods